docs/goodtoknow.rst

```diff
@@ -43,3 +43,36 @@ Which function to use when resampling data
 	to go from daily data to coarser (monthly, seasonal, annual).
 
 What is currently not covered by either `xscen` or `xclim` is a method to resample from data coarser than daily, where the base period is non-uniform (ex: resampling from monthly to annual data, taking into account the number of days per month).
+
+
+Metadata translation
+--------------------
+
+xscen itself does not add many translatable attributes, but when it does, it will look into xclim's options for which locales to translate them to. In a config file, activating French translations for both xclim's indicators and xscen is done with :
```

Comment:
maybe add little more about how that works for the people who are less familiar with xclim. i.e. It creates a new attrs with  `_fr`.